packages/web/src/app/shared/component-documentation/ceg/controls/counter/counter.component.html

```diff
@@ -7,12 +7,8 @@
     [disabled]="counter.value <= counter.min || disabled"
   >
     <span class="e-btn__icon">
-      <i
-        class="e-icon e-icon--subtract_circle"
-        [ngClass]="{ 'e-icon--color-disabled': counter.value <= counter.min || disabled }"
-        aria-hidden="true"
-      ></i>
-      <i class="e-icon e-icon e-icon--subtract_circle-filled-color" aria-hidden="true"></i>
+      <e-icon name="subtractCircle" />
+      <e-icon name="subtractCircleFilledColor" />
```

Comment:
Is this one missing `[ngClass]="{ 'e-icon--color-disabled': counter.value <= counter.min || disabled }"`?

Reply:
Nope, it doesn't need it! The icon color propagates from `[disabled]` on the buttons!